### ISSUE TITLE
Default training still uses TimeBatch?

### ISSUE THREAD

**Issue by cengc13** [nayoung10/MOFFlow#2]
Hi, Thank you for the great work. 

I noticed that the training dataloader still calls the TimeBatcher like below:

```python3
def train_dataloader(self, rank=None, num_replicas=None):
        num_workers = self.loader_cfg.num_workers
        return DataLoader(
            self._train_dataset,
            batch_sampler=TimeBatcher(
                sampler_cfg=self.sampler_cfg,
                dataset=self._train_dataset,
                rank=rank,
                num_replicas=num_replicas,
            ),
            num_workers=num_workers,
            prefetch_factor=None if num_workers == 0 else self.loader_cfg.prefetch_factor,
            pin_memory=False,
            persistent_workers=True if num_workers > 0 else False,
        )
```

In the paper Appendix, it was mentioned that Batch version is way more speedy. Could you upload the updated training procedure using Batch? Did I misunderstand anything here? Thanks for your time.

**Comment by nayoung10** [comment]
Hi, thank you for your interest in our work! Have you checked out the `main` branch (or the `batch` tag)? It should look like this

`    def train_dataloader(self, shuffle=True):
        batch_size = self.loader_cfg.batch_size.train
        num_workers = self.loader_cfg.num_workers
        return DataLoader(
            self._train_dataset,
            shuffle=shuffle,
            batch_size=batch_size,
            num_workers=num_workers,
            prefetch_factor=None if num_workers == 0 else self.loader_cfg.prefetch_factor,
            pin_memory=False,
            persistent_workers=True if num_workers > 0 else False,
        )
`

**Comment by cengc13** [comment]
Thank you for your quick response. My problem---I cloned the repo more than 2 months ago and I forgot to sync your updates.
